src/main/java/org/folio/rest/impl/LoadHoldingsImpl.java

```diff
@@ -61,13 +63,19 @@ public void postLoadHoldings(String contentType, Map<String, String> okapiHeader
   }
 
   @Override
-  public void getLoadHoldingsStatus(String contentType, Map<String, String> okapiHeaders,
+  @Validate
+  @HandleValidationErrors
+  public void getEholdingsLoadingKbCredentialsStatusById(String id, String contentType, Map<String, String> okapiHeaders,
                                     Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
     logger.info("Getting holdings loading status");
-    String tenantId = TenantTool.tenantId(okapiHeaders);
     RMAPITemplate template = templateFactory.createTemplate(okapiHeaders, asyncResultHandler);
     template
-      .requestAction(context -> holdingsStatusRepository.findByCredentialsId(context.getCredentialsId(), tenantId))
+      .requestAction(context ->
+        holdingsStatusRepository.findByCredentialsId(context.getCredentialsId(), context.getOkapiData().getTenant()))
+      .addErrorMapper(NotAuthorizedException.class, error401NotAuthorizedMapper())
+      .addErrorMapper(NotFoundException.class,
+        e -> GetEholdingsLoadingKbCredentialsStatusByIdResponse.respond404WithApplicationVndApiJson(
+          ErrorUtil.createError(e.getMessage())))
```

Comment:
```suggestion
      .addErrorMapper(NotFoundException.class, ExceptionMappers.error404NotFoundMapper())
```
You can replace this custom creating response code with the existing exception mapper.